README.rst

```diff
@@ -169,8 +169,22 @@ returns any status code besides 200, 4xx, or 5xx, this also becomes an
 Finally, if the web service returns a 200 but the body is invalid, the client
 throws a ``GeoIP2Error``.
 
+Database Usage
+--------------
+
+To use the database API, you first construct a ``geoip2.database.Reader`` using
+the path to the file as the first argument. After doing this, you may call the
+method corresponding to request type (e.g., ``city`` or ``country``), passing it
+the IP address you want to look up.
+
+If the request succeeds, the method call will return a model class for the
+end point you called. This model in turn contains multiple record classes,
```

Comment:
Is endpoint the term we want to use here? It seems perhaps a little odd in the context of the database. Likewise with "request" in this section.